exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -114,11 +117,12 @@
    * @param context Bootstrap context.
    * @param workManager WorkManager instance.
    */
-  public WebServer(final BootStrapContext context, final WorkManager workManager) {
+  public WebServer(final BootStrapContext context, final WorkManager workManager, final Drillbit drillbit) {
```

Comment:
Do we really want to pass the entire Drillbit to the web server? Seems like this creates a coupling that is too tight.

What services are needed? Only the shutdown? Something else?

Can we define an interface just for those services, and let `Drillbit` implement that interface to avoid the tight coupling?